lib/libc/musl/libc.s

```diff
@@ -0,0 +1,5089 @@
+y0f:
+.globl y0f
+.type y0f, @function;
+copy_file_range:
+.globl copy_file_range
+.type copy_file_range, @function;
+strlen:
+.globl strlen
+.type strlen, @function;
```

Comment:
Does order matter here? This seems to work but I've just realized that for glibc we use the following order:
```s
.globl strlen
.type strlen, @function;
strlen:
```
Also I just noticed that `recvmsg` is cut off at the end, this will be fixed shortly with the generator tool.

Reply:
Changed the order to match the glibc stubs for consistency and fixed the `recvmsg` stub.

Reply:
No, order does not matter. Probably sorted would be nicest so that our diffs are smaller if and when musl updates.